PlayerCore/components/New VRM Core/ScheduledVRMItems.swift

```diff
@@ -4,16 +4,33 @@
 import Foundation
 
 public struct ScheduledVRMItems {
-    static let initial = ScheduledVRMItems(items: [])
+    static let initial = ScheduledVRMItems(items: [:])
     
-    public let items: Set<VRMCore.Item>
+    public struct Candidate: Hashable {
+        public let id = VRMCore.ID<Candidate>()
+        public let source: VRMCore.Item.Source
+    }
+    
+    public var items: [VRMCore.Item: Set<Candidate>]
 }
 
 func reduce(state: ScheduledVRMItems, action: Action) -> ScheduledVRMItems {
     switch action {
     case let startGroupAction as VRMCore.StartGroupProcessing:
-        let allScheduledItems = state.items.union(startGroupAction.group.items)
-        return ScheduledVRMItems(items: allScheduledItems)
+        var newState = state
+        startGroupAction.group.items.forEach { item in
+            if newState.items[item] == nil {
```

Comment:
A lot of optionals and too complex - what you are trying to achieve here?
I think this can be simplified.

Reply:
Hmm, I'm agree it could be simplified.
We don't need this check `newState.items[item] == nil` because it's always `true` for new group